modules/installation-mirror-repository.adoc

```diff
@@ -174,6 +174,11 @@ $ oc image mirror -a ${LOCAL_SECRET_JSON} --from-dir=${REMOVABLE_MEDIA_PATH}/mir
 ----
 +
 <1> For `REMOVABLE_MEDIA_PATH`, you must use the same path that you specified when you mirrored the images.
++
+[IMPORTANT]
+====
+Due to the inclusion of old images in some image indexes, running `oc image mirror` might result in the following error: `error: unable to retrieve source image`. As a temporary workaround, you can use the `--skip-missing` option to bypass the error and continue downloading the image index. For more information, see link:https://access.redhat.com/solutions/6975305[Service Mesh Operator mirroring failed]. 
```

Comment:
Is **inclusion** the right word here. 
I get what your trying to say; we 'included' something we shouldn't have, but I am not 100% sure that will be clear to the reader who isn't well versed in what causes this issue. 

Is the following more informative (or too wordy)? 
```
Some image indexes include image references to images that no longer exist (on the registry). These images may have been pulled for various reasons, however the image index includes them; primarily to allow customers running those images an upgrade path off of that point to newer points on the upgrade graph, thus its possible that those mirroring some image indexes (with `oc image mirror`) will need to use `--skip-missing` images to avoid `error: unable to retrieve source image` errors, seen by these situations. For more information, see link:https://access.redhat.com/solutions/6975305[Service Mesh Operator mirroring failed].
```